.golangci.yml

```diff
@@ -73,6 +73,9 @@ issues:
     - path: routers/routes/routes.go
       linters:
         - dupl
+    - path: routers/api/v1/repo/issue.go
+      linters:
+        - dupl
```

Comment:
Why have you added this?

Reply:
revive told my via CI that `DelIssueSubscription` and `AddIssueSubscription` is the same :(

-> I'll test it again and we will see ...

Reply:
@zacheryph look at https://drone.gitea.io/go-gitea/gitea/14986/1/10 :(

Reply:
Ok put it back - generally if we get those dupl warnings we should refactor but it seems like that one is including comments which is particularly annoying and can't be refactored away.